contracts/pools/aeth/RateCalculatorAETH.vy

```diff
@@ -0,0 +1,24 @@
+# @version 0.2.11
+"""
+@title Curve Pool Rate Calculator 
+@author Curve.Fi
+@license Copyright (c) Curve.Fi, 2021 - all rights reserved
+@notice Logic for calculating exchange rate for a pool
+@dev This contract is only a template, pool-specific logic
```

Comment:
This comment can be removed in the not-template version

Reply:
oh shooot def forgot to remove that will update